Update auth.mdx to include important note about local testing (#32946)
supabase/supabase: apps/docs/content/guides/functions/auth.mdx

})
```

Important notes to consider:
* This is done _inside_ the `Deno.serve()` callback argument, so that the Authorization header is set for each request.

> * If you plan to test locally with `supabase functions serve` you _must_ use `Deno.env.get('SUPABASE_URL')`; `http://localhost:54321` will fail because of docker containerization – see [github discussion](https://github.com/orgs/supabase/discussions/32437#discussioncomment-11706231) for more info.

Reviewer: ```suggestion
* Use `Deno.env.get('SUPABASE_URL')` to get the URL associated with your project. Using a value such as `http://localhost:54321` for local development will fail due to Docker containerization.
```

## Integrating with Supabase Auth

Important notes to consider:

> * This is done _inside_ the `Deno.serve()` callback argument, so that the Authorization header is set for each request.

Reviewer 2: ```suggestion
- This is done _inside_ the `Deno.serve()` callback argument, so that the Authorization header is set for each request.
```

> * Use `Deno.env.get('SUPABASE_URL')` to get the URL associated with your project. Using a value such as `http://localhost:54321` for local development will fail due to Docker containerization.

Reviewer 2: ```suggestion
- Use `Deno.env.get('SUPABASE_URL')` to get the URL associated with your project. Using a value such as `http://localhost:54321` for local development will fail due to Docker containerization.
```
